README.md

```diff
@@ -1,2 +1,199 @@
-# Bookstore
-Bookstore app that allow user can display a list a books, add a new book, and remove book. Built with React, Redux, and external API
+<div align="center">
+
+  <h3><b>Book store App</b></h3>
+
+</div>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Book store App] <a name="about-project"></a>
+
+**[Book store App]** The Bookstore app that allow user can display a list a books, add a new book, and remove book. Built with React, Redux, and external API
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This app bulit with React, Redux and API
+<details>
+  <summary>Structure</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Layout</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/css">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li><a href="https://www.stylelint.io/">stylelint</a></li>
+    <li><a href="https://www.eslint.org/">eslint</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>React</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The following are the key features of the application.
+
+- [ ]  Linters setup
+- [ ]  React setup
+- [ ]  Initialize Bookstore app with components (React only)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [optional] I Advise you remove the `livedemo `section and add it back when your project has a deployed link. This will make your README clean and more professional.